src/sidebar/store/modules/selection.js

```diff
@@ -5,10 +5,26 @@
 
 /**
  * @typedef User
- * @property {string} userid - Unique user's id
+ * @property {string} [userid]
+ * @property {string} [username]
  * @property {string} displayName - User's display name
  */
 
+/**
+ * @typedef FocusedUser
+ * @property {string} filter - The identifier to use for filtering annotations
```

Comment:
What is the value of `filter` though? Is it _either_ a userid or a username, or something else derived from them?